coralproject/talk
# [CORL-2439]: Fix configurable flatten replies to ensure included in local state

## What does this PR do?

This PR updates to ensure that we are setting `flattenReplies` upon initialization of local state based on settings. This is for https://vmproduct.atlassian.net/browse/CORL-2439.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## Does this PR introduce any new environment variables or feature flags? 


## How do I test this PR?

Confirm that as you toggle Flatten replies on and off in the admin, it is reflected in the stream. Note that since you're in the development env, it is grabbing both flatten replies and feature flags from settings with local testing.

 
## How do we deploy this PR?

Ensure that flatten replies is set to on for any tenants that need it on.
